app/src/main/res/menu/menu_item_comment.xml

```diff
@@ -0,0 +1,14 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.menu.menu_item_comment` appears to be unused
 Unused resources make applications larger and slow down builds.